tensilelite/Tensile/Source/lib/include/Tensile/ContractionProblem.hpp

```diff
@@ -894,8 +894,24 @@ namespace Tensile
 
         PerformanceMetric performanceMetric() const
         {
-            const bool experimental = Debug::Instance().useExperimentalSelection();
-            return experimental ? PerformanceMetric::Experimental : m_performanceMetric;
+            const int experimental = Debug::Instance().useExperimentalSelection();
+            auto option       = static_cast<ExperimentalOption>(experimental);
```

Comment:
have you applied clang-format (.githooks/pre-commit)?

Reply:
Running .githooks/pre-commit doesn't seem to modify any files but when running directly from /opt/rocm/llvm/bin/clang-format -i on my files I see changes not limited to my commit. Please advise

Reply:
clang changes made as requesting. not limited to my modifications